index.html

```diff
@@ -7,33 +7,241 @@
+        </div>
+      </button>
+    </div>
+
+    <div>
+      <img class="img-laptop1" src="images/laptop-img4.png" alt="This is an image of a laptop." width="320" height="186">
+    </div>
+    <div class="section-dark bg-dark-blue">
+      <p class="project-goes white">Project name goes here</p>
+      <ul class="space">
+        <li class="greenish-box">HTML/CSS</li>
+        <li class="greenish-box">Ruby on Rails</li>
+        <li class="greenish-box">JavaScript</li>
+      </ul>
+      <button class="see-button bg-dark-blue white">
+        <div class="pointing">
+          <p>See this project</p>
+          <img class="icon" src="images/arrow.png" alt="This is a arrow pointing to the right icon" width="21" height="16">
+        </div>
+      </button>
+    </div>
+
+    <div>
+      <img class="img-laptop1" src="images/laptop-img5.png" alt="This is an image of a laptop." width="320" height="186">
+    </div>
+    <div class="section-dark bg-dark-blue">
+      <p class="project-goes white">Project name goes here</p>
+      <ul class="space">
+        <li class="greenish-box">HTML/CSS</li>
+        <li class="greenish-box">Ruby on Rails</li>
+        <li class="greenish-box">JavaScript</li>
+      </ul>
+      <button class="see-button bg-dark-blue white">
+        <div class="pointing">
+          <p>See this project</p>
+          <img class="icon" src="images/arrow.png" alt="This is a arrow pointing to the right icon" width="21" height="16">
+        </div>
+      </button>
+    </div>
+
+    <div>
+      <img class="img-laptop1" src="images/laptop-img6.png" alt="This is an image of a laptop and a phone." width="320" height="186">
+    </div>
+    <div class="section-dark bg-dark-blue">
+      <p class="project-goes white">Project name goes here</p>
+      <ul class="space">
+        <li class="greenish-box">HTML/CSS</li>
+        <li class="greenish-box">Ruby on Rails</li>
+        <li class="greenish-box">JavaScript</li>
+      </ul>
+      <button class="see-button bg-dark-blue white">
+        <div class="pointing">
+          <p>See this project</p>
+          <img class="icon" src="images/arrow.png" alt="This is a arrow pointing to the right icon" width="21" height="16">
+        </div>
+      </button>
     </div>
   </section>
   
+  <!--About Myself container-->
+  <section class="section-container">
+    <div class="section-sub">
+      <h1 class="title dark-blue">About</h1>
+
+      <div class="contents">
+        <p class="paragraph dark-blue">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        <button class="collab-button bg-dark-blue white">
+          <div class="cv-download">
+            <p>Get my CV</p>
+            <img class="icon" src="images/ic_download.png" alt="This is a download icon" width="24" height="18">
+          </div>
+        </button>
+      </div>
```

Comment:
Great job on your portfolio! I just wanted to point out two small things:

- [ ] Please double-check the spacing between your title and paragraph to make sure it looks consistent.👍
- [ ] Please **align the icon properly** inside your `Get my Resume` button to make it look more polished. 🙌

  ### Your design:
  ![image](https://user-images.githubusercontent.com/107952724/236647677-a57afa87-3419-4317-b736-01fa87797744.png)

  ### Figma design:
  ![image](https://user-images.githubusercontent.com/107952724/236647713-fa7f122e-a3e8-4fa5-9ad2-b4c42617069c.png)